.github/workflows/android-e2e-test.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         with:
           working-directory: ${{ env.WORKING_DIRECTORY }}
           api-level: 31
-          target: default
+          target: aosp_atd
```

Comment:
It seems like this change does the trick 🤔 

Actually you have "System UI isn't responding" popup instead of ANR (Application non responsive). So it's a different story, and from what observed in my experience "System UI isn't responding" happens if runner is weak. I've tried:
- to use macos-14 runner (but in this case the emulator doesn't start at all)
- to manually suppress this popup (doesn't work and popup is still present)
- to switch to ATD image (Automated Test Device) - and it seems like it really helped

However I still think that keeping test butler can help in the future to prevent undesired ANR and make e2e tests more reliable 👀